src/node/markdown/markdown.ts

```diff
@@ -19,7 +19,9 @@ import attrs from 'markdown-it-attrs'
 import emoji from 'markdown-it-emoji'
 import toc from 'markdown-it-toc-done-right'
 
-export type ThemeOptions = Theme | { light: Theme; dark: Theme }
+export type ThemeValue = Theme | IShikiTheme
```

Comment:
We probably can directly use `IThemeRegistration` here instead of defining `ThemeValue`.

Reply:
Thanks, fixed it!